config/locales/es.yml

```diff
@@ -0,0 +1,11 @@
+es:
+  ibandit:
+    invalid_country_code: "'%{country_code}' no es un código de país ISO 3166-1 válido"
+    invalid_check_digits: "Los dígitos de control no han pasado la verificación de módulo. Se esperaba: '%{expected_check_digits}', recibido '%{check_digits}'."
+    invalid_length: "La longitud no casa con la especificación SWIFT (Se esperaba %{expected_length} símbolos, se ha recibido %{length})"
+    is_required: "es obligado"
```

Comment:
I think "obligatorio" fits slightly better with what we mean. [This thread](http://forum.wordreference.com/showthread.php?t=1233290) seems to agree.